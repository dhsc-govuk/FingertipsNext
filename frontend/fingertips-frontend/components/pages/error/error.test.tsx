import { render } from '@testing-library/react';
import { expect } from '@jest/globals';
import { ErrorPage } from '.';

describe('Error Page', () => {
  it('snapshot test', () => {
    const container = render(
      <ErrorPage />
    );

    expect(container.asFragment()).toMatchSnapshot();
  });
});
