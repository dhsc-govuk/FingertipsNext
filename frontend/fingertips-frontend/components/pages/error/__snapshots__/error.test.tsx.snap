// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error Page > snapshot test 1`] = `
<DocumentFragment>
  <h1
    class="heading__StyledHeading-sc-1dsobun-0 eaMQId"
    data-testid="error-page-title"
  >
    Sorry, there is a problem with the service
  </h1>
  <div
    class="src__StyledParagraph-sc-sgc9po-0 qWnWb"
  >
    <p>
      Try again later.
    </p>
  </div>
  <div
    class="src__StyledParagraph-sc-sgc9po-0 qWnWb"
  >
    <p>
      You can 
      <a
        class="src__Link-sc-1loawqx-0 jjZtWC"
        href="/"
        node="[object Object]"
      >
        go back to the homepage
      </a>
       and start your search again.
    </p>
  </div>
</DocumentFragment>
`;
